docs/cookbook/custom-theme.mdx

```diff
@@ -0,0 +1,185 @@
+# Custom Theme in widgetbook (WIP)
```

Comment:
I can't see this page in the menu. I know we discussed that we don't want to show it to users already right now but is there a way to preview it for me?